package.json

```diff
@@ -30,10 +30,10 @@
   },
   "prepublish": "npm run prepare",
   "scripts": {
-    "solc": "../gsn-sponsor/scripts/solc contracts/*.sol 2>&1",
-    "test": "yarn run lint && yarn run truffle-compile && yarn generate && yarn tsc && npm run test-js",
+    "test": "yarn run lint && yarn generate && yarn tsc && npm run test-js",
     "ganache": "ganache-cli --hardfork 'istanbul' --gasLimit 100000000 --defaultBalanceEther 1000 --deterministic --keepAliveTimeout 2147483647",
     "test-js": "run-with-testrpc --hardfork 'istanbul' --port 8544 --gasLimit 10000000 --defaultBalanceEther 1000 --deterministic 'npx truffle --network npmtest test'",
+    "test-bail": "run-with-testrpc --hardfork 'istanbul' --port 8544 --gasLimit 10000000 --defaultBalanceEther 1000 --deterministic 'npx truffle --network npmtest test --bail'",
```

Comment:
why?

Reply:
for testing, it is sometimes better to stop after the first failure - instead of waiting 3-5 minutes for all tests to finish - before seeing the error reason...
(If I could see the stack trace of the already failed error, I wouldn't mind. but truffle (mocha) keeps you in suspense, and show everything at the end..

Reply:
oh, cool. we should run this in CI as well.